components/tabs/stories/tabs.test.js

```diff
@@ -18,30 +35,61 @@ export const TabsGroups = Variants({
 			testHeading: "Quiet",
 			isQuiet: true,
 		},
+		{
+			testHeading: "Quiet emphasized",
+			isQuiet: true,
+			isEmphasized: true,
+		},
 		{
 			testHeading: "Compact",
 			isCompact: true,
+			isQuiet: true,
+		},
+		{
+			testHeading: "Compact emphasized",
+			isCompact: true,
+			isQuiet: true,
+			isEmphasized: true,
+		},
+		{
+			testHeading: "Vertical tabs",
+			orientation: "vertical",
+		},
+		{
+			testHeading: "Emphasized vertical tabs",
+			orientation: "vertical",
+			isEmphasized: true,
+		},
+		{
+			testHeading: "Compact vertical tabs",
+			orientation: "vertical",
+			isCompact: true,
+			isQuiet: true,
+		},
+		{
+			testHeading: "Right vertical tabs",
+			orientation: "vertical",
+			hasRightAlignedTabs: "true",
+		},
+		{
+			testHeading: "Overflow",
```

Comment:
![image](https://github.com/user-attachments/assets/720d04aa-cce0-4cde-bc38-ba02bb0ef245)

I noticed this in the testing preview, should the line for Overflow Label only look like that? (If not, let's make it a separate issue because that feels like it might be a CSS change!)

Reply:
Hmmm, I'm not sure. Personally, I think it _looks_ like a bug. And we do set the `maxInlineSize` to 50px in the template, so it might not need CSS, but...I think your screenshot is actually more akin to what we see on the docs site for overflow tabs. The 50px is also set inline on the docs site for their examples as well.

<img width="300" alt="Screenshot 2024-09-13 at 11 45 56 AM" src="https://github.com/user-attachments/assets/7ef92d3a-41e8-4530-85fd-bc148b14dc8d">
<img width="445" alt="Screenshot 2024-09-13 at 11 45 47 AM" src="https://github.com/user-attachments/assets/f8d68252-f563-4276-9b69-72123de59866">